### PR TITLE
Add auth to install instructions

### PR DESCRIPTION
**Pull request by bduggan**
This change adds the `auth` portion to the installation instructions.  Currently, running `zef install Jupyter::Kernel` finds `Jupyter::Chatbook` before this one; since `Jupyter::Kernel` is also in the `provides` section of the [META6.json](https://github.com/antononcube/Raku-Jupyter-Chatbook/blob/master/META6.json#L31) for that module and the precedence in zef is a little unclear.

Not sure if there's a better way to handle this .. @antononcube, any ideas?  I think ideally Chatbook would be in a separate namespace -- this would also make it easier for both installations to exist at once on the same host, and it would be easier to choose one of the two kernels from within jupyter.

### CODE REVIEW

**Comment by antononcube** [comment]
I can rename the ['Kernel directory"](https://github.com/antononcube/Raku-Jupyter-Chatbook/tree/master/lib/Jupyter/Kernel) and corresponding rakumod file of "Jupyter::Chatbook" to "Chatbook". 

I kept the original names for reference purposes. (And as an acknowledgment...)

**Comment by bduggan** [comment]
Thanks for the acknowledgement :-). Ideally this could be handled by `zef` somehow, perhaps with versions...though I think the current state is that the name resolution during installation is not well defined.

**Comment by antononcube** [comment]
I renamed the [code base to of "Jupyter::Chatbook"](https://github.com/antononcube/Raku-Jupyter-Chatbook/tree/master) to use the namespace "Jupyter::Chatbook" instead of "Jupyter::Kernel" -- see [9184239](https://github.com/antononcube/Raku-Jupyter-Chatbook/commit/9184239c269fe3a3193a86a8e580bb243365f76a).

I also [uploaded](https://raku.land/zef:antononcube/Jupyter::Chatbook) to Zef ecosystem the new version of "Jupyter::Chatbook".


------


I think it is consistent because of that namespace renaming to also change the Jupyter kernel spec to indicate that "RakuChatbook" kernel is being used (see the screenshot), so, I made the corresponding code refactoring and changes.

A few related points / TODOs:
 
- I will write a more detailed description in the README. 
    - Basically, I have to copy and "localize" the setup instructions of "Jupyter::Kernel".

- It would be nice to be able to choose between a regular kernel and a chatbook kernel. 
    - I might need some help or directions about the having the proper implementation and setup instructions.

**Comment by bduggan** [comment]
Nice, thanks!

**Comment by bduggan** [comment]
Also, if there are any changes here that would allow the Chatbook classes to use the Kernel classes via regular inheritance (or some other OO mechanisms), I'm happy to incorporate them.
